release-notes.md

```diff
@@ -1,5 +1,40 @@
 # Two-Node Toolbox Release Notes
 
+## Version 0.5.2 - KCLI Deployment Integration
+*Release Date: August 2025*
+
+### New Features
+
+#### KCLI Alternative Deployment Method
+- Added `kcli-install` role providing alternative deployment method using kcli virtualization management tool
+- Support for both fencing and arbiter topologies with kcli-based deployment
+- Automated libvirt virtualization stack and kcli installation from COPR repository
+- Integration with kcli's BMC/Redfish simulation for fencing configuration
+- Pacemaker fencing configuration available via `kcli-redfish.yml` playbook for TNF clusters
+
+#### Modular Proxy Setup
```

Comment:
Does the proxy stuff work regardless of deployment type? i.e. kcli & devscripts?